mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/AssemblyMirror.cs

```diff
@@ -15,8 +16,13 @@ public class AssemblyMirror : Mirror
 		AssemblyName aname;
 		AssemblyDefinition meta;
 		AppDomainMirror domain;
+		byte[] metadata_blob;
+		bool? isDynamic;
+		byte[] pdb_blob;
 		Dictionary<string, long> typeCacheIgnoreCase = new Dictionary<string, long> (StringComparer.InvariantCultureIgnoreCase);
 		Dictionary<string, long> typeCache = new Dictionary<string, long> ();
+		Dictionary<uint, long> tokenTypeCache = new Dictionary<uint, long>();
+		Dictionary<uint, long> tokenMethodCache = new Dictionary<uint, long>();
```

Comment:
Please use the mono coding conventions i.e. space before '(' both in the c# and c code.